osm2lanes/src/osm.rs

```diff
@@ -4,52 +4,3 @@ pub use osm_reader::{NodeID, OsmID, RelationID, WayID};
 
 // This is a commonly used key in the codebase, so worthy of a bit of typo-prevention
 pub const HIGHWAY: &str = "highway";
-
-#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Debug)]
```

Comment:
All unused in this codebase and the list of highway types has become out-of-date with everything now. So removing.